webapp/graphql/PostQuery.js

```diff
@@ -128,3 +128,51 @@ export const PostsEmotionsByCurrentUser = () => {
     }
   `
 }
+
+export const relatedContributions = i18n => {
+  const lang = i18n.locale().toUpperCase()
+  return gql`query Post($slug: String!) {
+    Post(slug: $slug) {
+      id
+      title
+      tags {
+        id
+      }
+      categories {
+        id
+        name
+        icon
+      }
+      relatedContributions(first: 2) {
+        id
+        title
+        slug
+        contentExcerpt
+        shoutedCount
+        categories {
+          id
+          name
+          icon
+        }
+        author {
+          id
+          name
+          slug
+          avatar
+          contributionsCount
+          followedByCount
+          followedByCurrentUser
+          commentedCount
+          location {
+            name: name${lang}
+          }
+          badges {
+            id
+            icon
+          }
+        }
+      }
+      shoutedCount
+    }
+  }`
+}
```

Comment:
never seen this before, not clear what it does... returns two relatedContributions, do you have a link to some docs on this?

Reply:
oh, sorry I can see it was already in the code base...

Reply:
Yes, it's one of the bugfixes along the way. There was `commented` requested on type `Post` which does not make a lot of sense. To reduce the size of the queries, we could potentially use fragments.